Multisig/UI/App/UIKit/Transaction Detail Cells/ConfirmationConfirmedPiece.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ConfirmationConfirmedPiece.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 02.12.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class ConfirmationConfirmedPiece: UINibView {
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var addressInfoView: AddressInfoView!
+    @IBOutlet private weak var verticalBar: UIView!
+
+    override func commonInit() {
+        super.commonInit()
+        titleLabel.setStyle(GNOTextStyle.body.color(.gnoHold))
+        NSLayoutConstraint.activate([
+            heightAnchor.constraint(equalToConstant: 88)
```

Comment:
why not in Nib? I thought you was for making as much as possible in Nibs 😜

Reply:
Because this piece view is used to construct the confirmations view programmatically, I need to set the height constraints for it. And because it is used programmatically, I need to set the constraint in code - the XIB does not allow to set height constraint for the File Owner view